undertow/src/main/java/org/wildfly/extension/undertow/UndertowSubsystemModel.java

```diff
@@ -18,8 +18,9 @@ public enum UndertowSubsystemModel implements SubsystemModel {
     VERSION_12_0_0(12), // WildFly 27
     VERSION_13_0_0(13), // WildFly 28
     VERSION_14_0_0(14), // WildFly 32-present
```

Comment:
Please update this line as well.

```suggestion
    VERSION_14_0_0(14), // WildFly 32-36
```